### ISSUE TITLE
"not a valid color 3 or 6 hexadecimals numbers" incorrect checker

### ISSUE THREAD

**Issue by itsgroup-co-nz** [w3c/css-validator#489]
Hi there,

The CSS hex checker is returning false-positive results, for example:

"#3c9c4b is not a valid color 3 or 6 hexadecimals numbers."
"#d4331c is not a valid color 3 or 6 hexadecimals numbers."
"#0d9488 is not a valid color 3 or 6 hexadecimals numbers."
"#f6f6f6 is not a valid color 3 or 6 hexadecimals numbers."

The above are all valid within the A-F 0-9 hex range, and being flagged as invalid.

Not a major issue, but good to fix.

**Comment by ylafon** [comment]
```
div {
color: #d4331c;
}
```
is valid, do you have a test case to reproduce the errors?

**Comment by ylafon** [comment]
Can't reproduce, please reopen if you see this again and please provide an example (live or a test case) so that it ca be debugged.
